packages/core/src/plugins/codeBlockCopyButtons.ts

```diff
@@ -45,9 +46,9 @@ const copyCodeBlockScript = `<script>
     }
     </script>`;
 
-module.exports = {
+export = {
   getScripts: () => [copyCodeBlockScript],
-  processNode: (pluginContext, node) => {
+  processNode: (_: any, node: cheerio.Element & DomElement) => {
```

Comment:
one small question: is `pluginContext` marked as `any` here because it's a user defined json and hard to type?

Reply:
I marked it as `any` because it's not used anywhere in the function. Not sure if `_:any` or `_: PluginContext` would be better?

Reply:
why not just use `_:PluginContext` since we're only taking that in anyways

Reply:
Thanks for the input guys, created a `PluginsContext` class in types.ts - let me know what you think